api/rest/restapi.go

```diff
@@ -666,6 +678,41 @@ func (api *API) unpinHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (api *API) pinPathHandler(w http.ResponseWriter, r *http.Request) {
+	var ci types.CidSerial
```

Comment:
we don't need a special type for CidSerial. It's a `string`. In any case, this RPC method should return `PinSerial` (in a sense, the result of resolving PinPath is PinSerial), and the API endpoint should return that.